### PR TITLE
Add frontend UI for Deleting a User

### PR DESCRIPTION
**Pull request by PineappleIOnic**
Based off the delete project form code

### CODE REVIEW

File: app/views/console/users/user.phtml [hunk 1]
```diff
@@ -165,7 +195,7 @@
                                         data-analytics-event="submit"
                                         data-analytics-category="console"
                                         data-analytics-label="Delete User Session"
-                                        data-service="users.deleteSession"
+                                        data-service="users.updateStatus"
```
Comment (eldadfux) [inline app/views/console/users/user.phtml:198]: Any reason you changed this action?

File: app/views/console/users/user.phtml [hunk 2]
```diff
@@ -93,6 +93,36 @@
...
                         </div>
+
+                        <h3 class="text-danger">Danger Zone</h3>
+
+                        <div class="box danger">
+                            <p>This is the area where you can delete this account.</p>
+                        
+                            <p>By deleting this account you will lose all data associated with this account.</p>
+                        
+                            <p>PLEASE NOTE: Account deletion is irreversible.</p>
```
Comment (eldadfux) [inline app/views/console/users/user.phtml:104]: `User` and `Account` in Appwrite mean different things, lets change it to: `User deletion is irreversible`

File: app/views/console/users/user.phtml [hunk 3]
```diff
@@ -93,6 +93,36 @@
                                 No user preferences found
                             </div>
                         </div>
+
+                        <h3 class="text-danger">Danger Zone</h3>
+
+                        <div class="box danger">
+                            <p>This is the area where you can delete this account.</p>
```
Comment (eldadfux) [inline app/views/console/users/user.phtml:100]: `This is the area where you can delete this user.`

File: app/views/console/users/user.phtml [hunk 4]
```diff
@@ -93,6 +93,36 @@
...
+                        
+                            <p>By deleting this account you will lose all data associated with this account.</p>
```
Comment (eldadfux) [inline app/views/console/users/user.phtml:102]: Same here

File: app/views/console/users/user.phtml (hunk 1 above)
Comment (PineappleIOnic) [reply app/views/console/users/user.phtml:198]: Shouldn't have been, just of been a left over from when I was experimenting. I'll revert it back

**Comment by PineappleIOnic** [comment]
All previously mentioned issues should be fixed with the next push, Sorry didn't know about the account/user differentiation.  😅

File: app/views/console/users/user.phtml [hunk 5]
```diff
@@ -93,6 +93,36 @@
...
+                                data-analytics-event="submit"
+                                data-analytics-category="console"
+                                data-analytics-label="Delete User"
+                                data-service="users.deleteUser"
+                                data-event="submit"
+                                data-param-user-id="{{router.params.id}}"
+                                data-success="alert,trigger,redirect"
+                                data-confirm="Are you sure you want to delete this user?"
+                                data-success-param-alert-text="Deleted user successfully"
+                                data-success-param-trigger-events="users.update"
```
Comment (eldadfux) [inline app/views/console/users/user.phtml:116]: Probably missed that in the previous review. The event here should be `users.delete`

Comment (eldadfux) [reply app/views/console/users/user.phtml]: Fixed it.

**Comment by eldadfux** [comment]
🥳

**Pull request merged**